Dockerfile

```diff
@@ -19,6 +19,8 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/*
 
 # Build ImageMagick with WebP support
+COPY files/wand.patch /tmp/wand.patch
```

Comment:
Please add comments about

* What this patch does?
* Why this patch is needed?

to understand this later :koala: